src/classes/MergeService.cls

```diff
@@ -24,11 +26,21 @@ global with sharing class MergeService {
      */
     public MergeService(String sobjectName) {
         this.sobjectName = sobjectName;
+        this.sobjectNameOriginal = sobjectName;
+            
         this.concerns = [
             SELECT Id, FieldName__c, IsForceOverwrite__c
             FROM MergeConcern__mdt
             WHERE SobjectName__c = :sobjectName
         ];
+        // If it's person account switch the sobject name back to accounts
+        // by having them seperate it allows each to have seperate concerns saved in the custom metadata
+        // added by danhowellnz
+        if(sobjectName == 'PersonAccount'){
```

Comment:
I would declare a constant named `PERSON_ACCOUNT_SOBJECT_NAME` as follows and use the constant instead. I know in other places in the current code base string literals are used, but generally it's good practice to avoid using string literals in code.

Especially in this case the desire would be for developers using this package to create merge concerns with a very specific, prescribed object name, in which case declaring a constant would make this requirement much more explicit.